Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
Not too happy with the single quotes we have banned almost everywhere else, but oh well …